web_group_expand/README.rst

```diff
@@ -0,0 +1,58 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Group Expand Buttons
```

Comment:
Could you add an upper "decoration" too? See https://github.com/OCA/maintainer-tools/pull/109